lldb/tools/lldb-dap/DAP.h

```diff
@@ -290,7 +290,8 @@ struct DAP {
   ///   \b false if a fatal error was found while executing these commands,
   ///   according to the rules of \a LLDBUtils::RunLLDBCommands.
   bool RunLLDBCommands(llvm::StringRef prefix,
-                       llvm::ArrayRef<std::string> commands);
+                       llvm::ArrayRef<std::string> commands,
+                       bool echo_commands = true);
```

Comment:
Should we make the default false? 
```suggestion
                       bool echo_commands = false);
```